src/BaseWrapper.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
We need to use licenses consistently, and copy the right one to LICENSE.md.

GPL or MIT?

Reply:
MIT I guess?